R/arith.R

```diff
@@ -63,6 +63,9 @@ vec_arith.logical.logical <- function(op, x, y) vec_arith_base(op, x, y)
 #' @method vec_arith.logical numeric
 #' @export
 vec_arith.logical.numeric <- function(op, x, y) vec_arith_base(op, x, y)
+#' @method vec_arith.logical integer
+#' @export
+vec_arith.logical.integer <- function(op, x, y) vec_arith_base(op, x, y)
```

Comment:
Are you sure this is necessary?

Reply:
Removed. I added it for symmetry when I needed `vec_arith.integer` for `integer64`. Obsolete now.